src/main/scala/com/github/unchama/seichiassist/menus/stickmenu/FirstPage.scala

```diff
@@ -266,31 +266,41 @@ object FirstPage extends Menu {
     }
 
     val computeRegionMenuButton: IO[Button] = IO {
-      val buttonLore = {
+      val (buttonLore, effect) = {
         val worldGuardPlugin = ExternalPlugins.getWorldGuard
-        val regionManager = worldGuardPlugin.getRegionManager(getWorld)
+        val regionManager = WorldGuardWrapper.getRegionManager(getWorld)
 
-        val maxRegionCount = WorldGuardWrapper.getMaxRegionCount(player, getWorld)
-        val currentPlayerRegionCount =
-          regionManager.getRegionCountOfPlayer(worldGuardPlugin.wrapPlayer(player))
+        regionManager.fold(
+          (
+            List(s"${GRAY}このワールドでは土地の保護は行なえません"),
+            LeftClickButtonEffect.apply(TargetedEffect.emptyEffect)
```

Comment:
`emptyEffect`をimportした上で、
```suggestion
            LeftClickButtonEffect(emptyEffect)
```
とすることができます。